numba/tests/test_remove_dead.py

```diff
@@ -78,8 +100,91 @@ def seed_call_exists(func_ir):
             return False
 
         test_ir = compiler.run_frontend(call_np_random_seed)
-        remove_dead(test_ir.blocks, test_ir.arg_names)
+        remove_dead(test_ir.blocks, test_ir.arg_names, test_ir)
         self.assertTrue(seed_call_exists(test_ir))
 
+    def test_alias_ravel(self):
```

Comment:
Seems like `test_alias_ravel`, `test_alias_flat`, `test_alias_transpose1`, `test_alias_reshape1`, and `test_alias_reshape2` could be combined/share code? (Also, why `transpose1`, there's no `transpose2`?, should that be present and check `np.transpose`?)

Reply:
done.